Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM debian:bullseye
```

Comment:
I don't know if it's intentional, but the last debian version is bookworm.

Reply:
indeed... released on February 10th, 2024.
wasn't aware

Reply:
Tested and still work, updated.

Thanks :+1: